### PR TITLE
Add UI for transaction filters

### LINKED ISSUE #558: Transactions spliting Received-sent
**People should be able to see a split view of their transactions.**
while standing on the Activity Log screen on the "transactions" tab, people should be able to choose to display only the received or only the sent transaction - apart from the already existing overall view order by date. This is helpful for business to be able to track better the payments the receive. 

**Design Material**
https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?type=design&node-id=9427-16806

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

File: src/components/ActivityStream.js [hunk 1]
```diff
@@ -167,6 +180,22 @@ const ActivityStreamList = ({ activities, lastSeenAt, lastUpdatedAt }) => {
             return acc;
           }
 
+          const info =
+            data.from === safeAddress
+              ? { prefix: '-', type: 'SENT' }
+              : { prefix: '+', type: 'RECEIVED' };
+
```
Comment (juanenrisley) [inline src/components/ActivityStream.js:187]: Both `if` statements could be in the same one as both are used for returning the same value

Comment (louilinn) [reply src/components/ActivityStream.js:187]: With an OR || between statements on line 189-190 and line 195, improve readability :)

**Review by louilinn — CHANGES_REQUESTED** [review]
Small code comment, otherwise great. 

One consideration is that when filtering and there is only one or maybe no received transaction the log will have 1 or no items and load more, rather than 10. We could trigger another call but I think we should only act on this behaviour is product brings it up. I see this as good enough :)

**Comment by louilinn** [comment]
Another thing we noticed: When switching to Received or Sent the title of the drop down should change accordingly

**Comment by mikozet** [comment]
> Another thing we noticed: When switching to Received or Sent the title of the drop down should change accordingly

it is corrected now

**Comment by juanenrisley** [comment]
@mikozet i see that the tests werent successful, maybe is something temporary. Could they be relaunched manually?

**Review by juanenrisley — APPROVED** [review]
👍🏿

**Review by louilinn — APPROVED** [review]

**Pull request merged**